tests/test_core.py

```diff
@@ -5740,9 +5740,8 @@ def do_autodebug_post(self, filename):
     Building.llvm_as(filename)
     Building.llvm_dis(filename)
 
-  # Broken on V8 but not node
-  @no_wasm_backend()
   def test_autodebug(self):
+    self.banned_js_engines = [V8_ENGINE] # Broken on V8 but not node; wasm-only
```

Comment:
if this is wasm-only, then this could be `if self.is_wasm():` and then your new line. then we'd still test v8 on the asm.js version.